.github/workflows/pkgci_regression_test.yml

```diff
@@ -115,7 +115,7 @@ jobs:
             -rpfE \
             --capture=no \
             --log-cli-level=info \
-            --timeout=600 \
+            --timeout=1200 \
```

Comment:
This timeout shouldn't need to be changed. Cache misses are likely most of the time increase.